demo_modules/shifting_shapes/shifting_shapes.ts

```diff
@@ -0,0 +1,216 @@
+/* Copyright 2023 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import { Client } from '../../client/modules/module_interface.ts';
+import { P5Canvas, P5Surface } from '../../client/surface/p5_surface.ts';
+import { Surface } from '../../client/surface/surface.ts';
+import { Polygon } from '../../lib/math/polygon2d.ts';
+
+export function load(wallGeometry: Polygon) {
+  const frames = 200;
+  const delayFrames = 400;
+  const amtGridx = 26;
+  const amtGridy = 10;
+  const amtShapes = 250;
+  const points = [];
+  const movers = [];
+  let nOff = 0;
+  const ease = {
+    easeInOutQuart: (t) =>
+      t < 0.5 ? 8 * t * t * t * t : 1 - 8 * --t * t * t * t,
+    easeOutQuint: (t) => 1 + --t * t * t * t * t,
+  };
+  const colors = [
+    '#BF0413',
+    '#023E73',
+    '#02733E',
+    '#F29F05',
+    '#F24405',
+    '#000',
+  ];
+
+  class Mover {
+    constructor(baseSize, p5) {
+      this.p5 = p5;
+      const getRandomPoint = ShiftingShapesSketch.getRandomPoint;
```

Comment:
const {getRandomPoint} = ShiftingShapesSketch works here, but...

... maybe you don't really need a static method, and a module-level function would be fine.